packages/template-retail-react-app/app/ssr.js

```diff
@@ -92,6 +101,32 @@ const {handler} = runtime.createHandler(options, (app) => {
         res.send()
     })
 
+    if (ENABLE_SSR_POST) {
+        app.post('/passwordless-login-callback', async (req, res) => {
```

Comment:
How might we stop baddies from these endpoints sending nefarious emails or exhausting marketing cloud sending quotas?

Reply:
Would a customer ever customize these endpoints? If so, how? If not, did you consider adding them as functions to an SDK such that folks could import them?

Reply:
We will be securing these POST endpoints through a separate ticket, linked [here](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000026RuyBYAS/view).